### PR TITLE
 Upgrade to 2-dev.68 and adjust Analysis options page

### PR DESCRIPTION
**Pull request by chalin**
Staged at https://sz-www-2.firebaseapp.com/guides/language/analysis-options#enabling-dart-2-semantics

### CODE REVIEW

File: src/_guides/language/analysis-options.md [hunk 1]
```diff
@@ -100,15 +101,17 @@ and `my_other_other_package`, and file #2 to analyze the code in
...
 {% endcomment %}
 
-The analyzer in the Dart SDK doesn't yet use the
-[latest Dart type system][sound-dart], by default.
-The simplest way to enable Dart 2 semantics is to put
+The analyzer, in Dart SDK releases 2.0.0-dev.68.0 and later,
+uses the [latest Dart type system][sound-dart] by default.
+
+The simplest way to enable Dart 2 semantics in earlier
+dev releases of the Dart 2 SDK is to put
```
Comment (kwalrath) [inline src/_guides/language/analysis-options.md]: "dev" isn't necessary here.

Comment (kwalrath) [reply src/_guides/language/analysis-options.md]: Below, please change "Performing additional type checks" from a 3rd-level heading to a 2nd-level heading.

File: src/_guides/language/analysis-options.md [hunk 2]
```diff
@@ -100,15 +101,17 @@ and `my_other_other_package`, and file #2 to analyze the code in
...
+## Enabling Dart 2 semantics (SDK \<2.0.0-dev.68.0) {#enabling-dart-2-semantics}
 
 {% comment %}
 update-for-dart-2
 {% endcomment %}
 
-The analyzer in the Dart SDK doesn't yet use the
-[latest Dart type system][sound-dart], by default.
-The simplest way to enable Dart 2 semantics is to put
+The analyzer, in Dart SDK releases 2.0.0-dev.68.0 and later,
```
Comment (kwalrath) [inline src/_guides/language/analysis-options.md]: Let's rewrite this slightly for clarity:

As of release 2.0.0-dev.68.0, [Dart 2 semantics][sound-dart] are on by default.
In earlier releases, the simplest way to enable Dart 2 semantics is to put...

[code snippet here]

Another way to enable Dart 2 semantics is to enable additional type checks.

## Enabling additional type checks

**Comment by chalin** [comment]
Updated. Staging server refreshed. PTAL.

**Pull request merged**